includes/Core/Admin/Screens.php

```diff
@@ -398,7 +398,7 @@ private function get_screens() {
 			array(
 				'title'            => __( 'User Input', 'google-site-kit' ),
 				'capability'       => Permissions::MANAGE_OPTIONS,
-				'parent_slug'      => null,
+				'parent_slug'      => defined( 'SHOW_USER_INPUT_SETTINGS_PAGE' ) ? Screen::MENU_SLUG : null,
```

Comment:
This should be removed, it should be `null`. We can always manually enter the URL for testing.

Reply:
Updated.